### PR TITLE
Ingest JEP/TALN/RECITAL 2017+2018

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by nschneid** [comment]
I see a lot of "[In French]" titles—wasn't the plan to mark the language in an XML attribute instead?

**Comment by mjpost** [comment]
Can you link to an example? I don't see this at all (even with grep).

File: data/xml/2017.jeptalnrecital.xml [hunk 1]
```diff
@@ -0,0 +1,676 @@
...
+      <publisher>ATALA</publisher>
+      <address>Orléans, France</address>
+      <month>6</month>
+      <year>2017</year>
+    </meta>
+    <frontmatter>
+      <url hash="ede7a90d">2017.jeptalnrecital-long.0</url>
+    </frontmatter>
+    <paper id="1">
+      <title>Ajout automatique de disfluences pour la synthèse de la parole spontanée : formalisation et preuve de concept (Automatic disfluency insertion towards spontaneous <fixed-case>TTS</fixed-case> : formalization and proof of concept)[In <fixed-case>F</fixed-case>rench]</title>
```
Comment (nschneid) [inline data/xml/2017.jeptalnrecital.xml:17]: [In French]

File: data/xml/2018.jeptalnrecital.xml [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+      <author><first>Thierry</first><last>Charnois</last></author>
+      <author><first>Bruno</first><last>Cremilleux</last></author>
+      <author><first>Albrecht</first><last>Zimmermann</last></author>
+      <pages>465–472</pages>
+      <abstract>Nous nous intéressons dans cet article à l’extraction de thèmes à partir de retranscriptions textuelles de réunions. Ce type de corpus est bruité, il manque de formatage, il est peu structuré avec plusieurs locuteurs qui interviennent et l’information y est souvent éparpillée. Nous présentons une étude expérimentale utilisant des méthodes fondées sur la mesure tf-idf et l’extraction de topics sur un corpus réel de référence (le corpus AMI) pour l’étude de réunions. Nous comparons nos résultats avec les résumés fournis par le corpus.</abstract>
+      <url hash="1e2c4383">2018.jeptalnrecital-court.30</url>
+    </paper>
+    <paper id="31">
+      <title>Analyse morpho-syntaxique en présence d’alternance codique (<fixed-case>P</fixed-case>o<fixed-case>S</fixed-case> tagging of Code Switching)[In <fixed-case>F</fixed-case>rench]</title>
+      <author><first>José</first><last>Carlos Rosales Núñez</last></author>
```
Comment (danielgildea) [inline data/xml/2018.jeptalnrecital.xml:434]: <first>José Carlos</first><last>Rosales Núñez</last>

File: data/xml/2017.jeptalnrecital.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/2017.jeptalnrecital.xml]: I fixed these with a Perl one-liner.

File: data/xml/2018.jeptalnrecital.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2018.jeptalnrecital.xml:473]: Fixed.

**Pull request merged**